### ISSUE TITLE
Deveroom init: FileNotFoundException: Could not load file or assembly System.Net.Http

### ISSUE THREAD

**Issue by gasparnagy** [specsolutions/deveroom-feedback#8]
Opening a feature file in Visual Studio (probably before VS 15.9.6) causes:

```
System.IO.FileNotFoundException: Could not load file or assembly &apos;System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. 
```

Upgrading to latest VS(?) and/or installing .NET SDK Version ??? solves the issue.

Full stack trace:
```
A MEF Component threw an exception at runtime: Microsoft.VisualStudio.Composition.CompositionFailedException: An exception was thrown while initializing part "Deveroom.VisualStudio.ProjectSystem.VsProjectSystem". --->
System.TypeInitializationException: The type initializer for 'Deveroom.VisualStudio.EventTracking.GoogleAnalyticsApi' threw an exception. ---> 
System.IO.FileNotFoundException: Could not load file or assembly 'System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified. 
at Deveroom.VisualStudio.EventTracking.GoogleAnalyticsApi..cctor() 
--- End of inner exception stack trace --- 
at Deveroom.VisualStudio.EventTracking.EventTracker.TrackEvent(String category, String action, String label, Nullable`1 value) 
at Deveroom.VisualStudio.EventTracking.EventTracker.TrackOpenProjectSystem(String vsVersion) 
at Deveroom.VisualStudio.ProjectSystem.VsProjectSystem..ctor(IServiceProvider serviceProvider, IVsPackageInstallerServices vsPackageInstallerServices, IVsSolutionEventListener solutionEventListener) in W:\Devero\Deveroom.VisualStudio\Deveroom.VisualStudio.Package\ProjectSystem\VsProjectSystem.cs:line 62 
--- End of inner exception stack trace --- 
at Microsoft.VisualStudio.Composition.RuntimeExportProviderFactory.RuntimeExportProvider.RuntimePartLifecycleTracker.CreateValue() 
at Microsoft.VisualStudio.Composition.ExportProvider.PartLifecycleTracker.Create() 
at Microsoft.VisualStudio.Composition.ExportProvider.PartLifecycleTracker.MoveNext(PartLifecycleState nextState) 
at Microsoft.VisualStudio.Composition.ExportProvider.PartLifecycleTracker.MoveToState(PartLifecycleState requiredState) 
at Microsoft.VisualStudio.Composition.ExportProvider.PartLifecycleTracker.GetValueReadyToExpose() at Microsoft.VisualStudio.Composition.RuntimeExportProviderFactory.RuntimeExportProvider.<>c__DisplayClass15_0.<GetExportedValueHelper>b__0()
```

**Comment by gasparnagy** [comment]
Seems to be reproducible with VS 15.8.6

**Comment by gasparnagy** [comment]
Fixed, the next patch release (1.0.2) will contain the fix
